packages/sdk-v2/src/sdk/analytics/amplitude/api.ts

```diff
@@ -0,0 +1,24 @@
+import { createInstance, Identify } from '@amplitude/analytics-browser';
+
+class Wrapper {
+  private api = createInstance()
+  readonly Identify = Identify
+
+  async initialize(apiKey: string) {
+    return this.api.init(apiKey);
+  }
+
+  setUserId(id: string): void {
+    this.api.setUserId(id);
+  }
+
+  setUserProperties(props: Record<string, unknown>) {
```

Comment:
1. `Record<string, any>`, not `unknown`
2. you  still have `object` in many places